public/coding/JavaScript/PokemonTemp.js

```diff
@@ -0,0 +1,307 @@
+const PokemonData = {
```

Comment:
It's overwhelming to have this pokemon data and the getFireblastFunction in the same file. Can we split them up into two different files? 

Eventually you are going to build a Coding Challenge UI that will look like this: https://codesandbox.io/s/holy-silence-oszdx?file=/src/App.js

So we will want to be able to support multiple file for a single coding challenge.